Python/compile.c

```diff
@@ -67,17 +67,38 @@
  */
 #define MAX_ALLOWED_STACK_USE (STACK_USE_GUIDELINE * 100)
 
+
+/* Pseudo-instructions used in the compiler,
+ * but turned into NOPs by the assembler. */
+#define SETUP_FINALLY 255
+#define SETUP_CLEANUP 254
+#define SETUP_WITH 253
+#define POP_BLOCK 252
```

Comment:
Is this a temporary situation?

Reply:
Maybe. I don't plan on changing it, but others might.
These are internal to the compiler. The rest of the code base shouldn't need to to care how we get from AST+symbols to bytecode.